packages/ui/src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx

```diff
@@ -41,6 +41,14 @@ const Mobile: React.FC<{
                   {x.amount?.displayDenom.toUpperCase()}
                 </Typography>
               </div>
+              <div className={classes.item}>
+                <Typography variant="h4" className="label">
+                  {t('commission')}
+                </Typography>
+                <Typography variant="body1" className="value">
+                  {x.commission ? `${x.commission} %` : ''}
```

Comment:
```suggestion
                  {numeral(x.commission).format('0%')}
```

Reply:
@teamchong wouldn't this `numeral` function converts the `(commission * 100) %` ? As my understanding is that the `commission` from hooks is already `*100 `: https://github.com/forbole/big-dipper-2.0-cosmos/blob/85bed3e38a198f5e52a6f99a7689de5d01fc2e5d/packages/ui/src/screens/account_details/components/staking/hooks.ts#L78)

http://numeraljs.com/#format

Reply:
@rachelhox yes, you are right